tests/ipsec/ipsec3hosts.pm

```diff
@@ -115,13 +123,14 @@ sub run ($self) {
         barrier_wait 'IPSEC_ROUTE_SETUP_CHECK_DONE';
         $self->config_ipsec($ipsec_setting_right);
         barrier_wait 'IPSEC_TUNNEL_MODE_SETUP_DONE';
-        assert_script_run("tcpdump -i $dev0 esp -c 4");
+        script_run("timeout 20 tcpdump -i $dev0");
```

Comment:
Why was `assert_script_run` changed to `script_run`?

Reply:
Once issue happen, the assert_script_run maybe trigger error but what i want is collect trace of tcpdump.